form_to_clean_csv.py

```diff
@@ -0,0 +1,23 @@
+import pandas as pd
+import sys
+
+# usage: [form.csv] [hw_number]
+# creates 2 files hw_number_handles.csv & hw_number_teams.csv
+
+data = pd.read_csv(sys.argv[1])
+df = pd.DataFrame(data)
+
+df = pd.DataFrame(data)
```

Comment:
I think `pd.read_csv()` returns a dataframe, so this can just become

```python
df = pd.read_csv(...)
```

and the next two lines can go away